### PR TITLE
VIH-8971 SDS Migration Files

### PR DESCRIPTION
**Pull request by chris-pateman**
### JIRA link (if applicable) ###
VIH-8971

### Change description ###
Add all pipeline and configuration for SDS migration

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: variables/staging.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+variables:
+  - name: env
+    value: stg 
+  - name: stagingTagName
+    value: "staging-$(Build.SourceVersion)"
```
Comment (v1ferrarij) [inline variables/staging.yaml]: this is wrong, it should be $(Build.BuildNumber)

File: variables/shared.yaml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    value: ${{ parameters.env }}.platform.hmcts.net
+
+  - name: gh_app_private_key_file_name
+    value: hmcts-ado-cnp-app.private-key.pem
+
+
+  - name: vhSQLServerName
+    value: vh-infra-core-${{ parameters.env }}
+  - name: UserApiDbName
+    value: vhbookings
```
Comment (v1ferrarij) [inline variables/shared.yaml]: Is this right?

Do multiple API's share the vhbookings db ?

File: charts/vh-user-api/values.dev.template.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+java:
+  image: '${IMAGE_NAME}'
+  ingressHost: ${SERVICE_FQDN}
+  releaseNameOverride: ${RELEASE_NAME}
+  environment:
```
Comment (v1ferrarij) [inline charts/vh-user-api/values.dev.template.yaml:6]: is this not missing the SERVICES__USERAPIRESOURCEID ?

File: charts/vh-user-api/values.demo.template.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+java:
+  image: '${IMAGE_NAME}'
+  ingressHost: ${SERVICE_FQDN}
+  releaseNameOverride: ${RELEASE_NAME}
+  environment:
```
Comment (v1ferrarij) [inline charts/vh-user-api/values.demo.template.yaml:6]: is this not missing the SERVICES__USERAPIRESOURCEID ?

File: azure-pipelines.sds.release.yml [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+              parameters:
+                addOrDelete: add
+                lockName: "${{ variables.env }}-lock"
+                resourceGroup: ${{ variables.vhResourceGroup }}
+                azureSubscription: ${{ variables.subscriptionName }}
+                lockType: CanNotDelete
+
+
+#####################################################
+# Approval Gate for Booking API Release. ############
```
Comment (v1ferrarij) [inline azure-pipelines.sds.release.yml]: Minor Issue - Comment still refer to Bookings

**Review by v1ferrarij — CHANGES_REQUESTED** [review]

**Review by v1ferrarij — APPROVED** [review]
LG

**Review by agovind92 — APPROVED** [review]

**Pull request merged**